src/plugins/local/selectors/localSelectors.js

```diff
@@ -37,6 +37,43 @@ export const metaDataColumnsSelector = dataSelectors.metaDataColumnsSelector;
 
 const columnPropertiesSelector = state => state.getIn(['renderProperties', 'columnProperties']);
 
+const substringSearch = (value, filter) => {
+  const filterToLower = filter.toLowerCase();
+  return value && value.toString().toLowerCase().indexOf(filterToLower) > -1;
+}
+
+const textFilterRowSearch = (row, columnProperties, filter) => {
+  return row.keySeq()
+    .some((key) => {
+      const filterable = columnProperties && columnProperties.getIn([key, 'filterable']);
+      if (filterable === false) {
+        return false;
+      }
+      return substringSearch(row.get(key), filter)
```

Comment:
I just noticed we're missing semicolons on most of the new code.

Reply:
Ah yeah - I tend to write mine without semi-colons. Might be worth adding an eslint or similar to ensure consistent styling